tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -87,5 +87,22 @@ setup_file() {
 
     # Check on manifest contents
     cat ~/.fvm/versions/0.10.15/manifest.json
-    assert_output --partial "{\"channel\":\"stable\",\"version\":\"0.10.15\"}"
+    assert_output --partial "{\"channel\":\"0.10.15\",\"version\":\"0.10.15\"}"
+}
+
+@test "fvm install straight installs `fluvio` stable" {
+    # Make sure `stable` is not installed already
+    ! test -d ~/.fvm/versions/stable
+    assert_success
+
+    run $FVM_BIN install
+    assert_success
+
+    # Check for `stable` dir to be present under "versions"
+    test -d ~/.fvm/versions/stable
+    assert_success
+
+    # Check on manifest contents
+    cat ~/.fvm/versions/stable/manifest.json
```

Comment:
why do you need manifest?

Reply:
The `manifest.json` file holds both:

- Tag
- Channel

When we want to know the tag behind a channel we use `manifest.json` to get such info.
This is also helpful for the `fvm show` command, which renders the list of installed Fluvio Versions.

We can use each `Manifest.json` to tell the tag for channel versions in such case as well.

An alternative could be keeping this data in `settings.toml`, like a list of installed versions with its tags, we would have to write against that on each `fvm install`.

Reply:
But you can scan directories for the installed version.  I am concerned that there are two sources of truth.

Reply:
Yeah I get you, I thought about doing the same at first, but then I noticed, how can I tell the version for a channel dir?

If I find `latest` or `stable` how can I tell the exact version number to tell if theres an update?

I thought that by having a manifest I could get that information.

Reply:
As just metadata, it is fine.